Odata-docs/client/v7/basic-crud-operations.md

```diff
@@ -33,6 +42,14 @@ foreach (var person in people)
     // business logic
 }
 ```
+or use the `Where()` with `First()` or `Single()` API call:
+
+```c#
+var person = context.People.Where(c => c.UserName == "russellwhyte").First();
+var person = context.People.Where(c => c.UserName == "russellwhyte").Single();
+```
+Note that `First()` will return one object, even if the lambda expression matches multiple objects.
+`Single()` always return one object and will throw an exception if the lambda expression matches multiple objects.
```

Comment:
I'd expect both `First()` and `Single()` will throw an exception if no person with `UserName` as `russelwhyte` exists?

Reply:
Yes. I have added that to the docs.
#Resolved